### PR TITLE
fix: windows start guide

### PR DESCRIPTION
**Pull request by delagroove**

### CODE REVIEW

File: guides/quick-start-windows.md [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+## Install WSL2
+
+Download it at the [official WSL microsoft site](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
   
 
 > **Note**: The automatic Windows Insider install comes with the Ubuntu distro. If you have manually installed WSL2, you can pick another Linux distro. This guide assumes you picked Ubuntu.
 
-### Install Docker Desktop WSL2 backend
+## Install Docker Desktop WSL2 backend
   - https://docs.docker.com/desktop/windows/wsl/
```
Comment (MohanNarayana) [inline guides/quick-start-windows.md]: Can we make this a sentence and embed the link in text. Something like "Download and install Docker Desktop WSL2 from [here](https://docs.docker.com/desktop/windows/wsl/)"

**Review by MohanNarayana — CHANGES_REQUESTED** [review]
One change for this

**Review by Akarshit — APPROVED** [review]

File: guides/quick-start-windows.md (hunk 1 above)
Comment (delagroove) [reply guides/quick-start-windows.md]: moving this to the other guide, shouldn't have 2

**Review by brent-hoover — APPROVED** [review]
This is a better approach than a separate Windows Quick Start. Do we have any way of testing any of this?

**Comment by delagroove** [comment]
@zenweasel it's copied verbatim from @rondlite 's guide. He had tested it.

**Comment by brent-hoover** [comment]
Closing to be reopened in a later phase

**Pull request closed without merging**